Resources/Prototypes/Catalog/Bounties/bounties.yml

```diff
@@ -247,7 +247,7 @@
 
 - type: cargoBounty
   id: BountyMonkeyCube
-  reward: 2000
+  reward: 800
```

Comment:
monkey cubes cost 2k for 6, even before this nerf you were breaking even on monkey cube bounties unless you got 2 in a row, now you're not even doing that.

Reply:
Chef restocks also come with monkey or at least random cubes and they only cost 500 or so, though minority of players make use of that.

Reply:
> monkey cubes cost 2k for 6, even before this nerf you were breaking even on monkey cube bounties unless you got 2 in a row, now you're not even doing that.

You can find monkey cubes in:

- virology at some stations
- chefvend thingy
- chef restocks

Reply:
virology is map dependent and bounties shouldn't rely on certain maps rolling to be profitable
chefvend, sure
chef restocks is edging onto exploit territory and should probably be PRd out.